.github/workflows/integration_tests.yml

```diff
@@ -106,10 +106,29 @@ jobs:
           path: results-${{github.run_id}}-${{matrix.label}}
           if-no-files-found: ignore  # this avoids a false "Warning" if there were no issues
 
+  playwright-tests:
+    runs-on: ubuntu-latest
+    needs: [notify-of-test-run-start, integration-tests]
+    env:
+      PLAYWRIGHT_BASE_URL: ${{ github.event.inputs.mozorg_service_hostname }}
+      CI: true
+      CI_JOB_ID: ${{ github.run_id }}
+      DRIVER: ""
+      LABEL: playwright-tests
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
+      with:
+        node-version: 20
+    - name: Install dependencies
```

Comment:
Open question: do I need to add `if: always()` to these steps?

>     # Note we use if: always() below to keep things going, rather than
>    # continue-on-error, because that approach falsely marks the overall
>    # test suite as green/passed even if it has some failures.

Reply:
No, I think don't it's worth adding them to here. 

What's not clear from my original comment that you quoted is that the need to keep going under failure is something specific to using a matrix strategy. Our preference (influenced by what we had in GitLab before, as well as practicaltiy) is that we don't stop running through the various test options set up in the matrix if something fails. By using `if: always()`, even if the tests fail under certain parameters, we get to see wholesale which do and which do not pass, right across the matrix, rather than failing fast and maybe having to re-run a few times to be shown all the bugs to be sorted.

Because your playwright tests don't use the matrix strategy, there's no need to worry about failing "too" fast.

Reply:
I can't suggest an inline change to the comment in this pr, but if you could change it to the following that'd be welcome

OLD
```
     # Note we use if: always() below to keep things going, rather than
    # continue-on-error, because that approach falsely marks the overall
    # test suite as green/passed even if it has some failures.
```
NEW
``` 
    # Note we use `if: always()` below to keep the entire matrix strategy going; 
    # We can't use `continue-on-error`, because that approach falsely marks 
    # the overall test suite as green/passed even if it has some failures.
```